### DISCUSSION TITLE
Prisma imported from `.prisma/client`

### DISCUSSION THREAD

**Discussion by cerpinn** [prisma/prisma#10374] [Q&A] [answered]
VSCode automatically imports prisma client from `.prisma/client`, instead of `@prisma/client`.

Do you guys have this issue too? any idea how to fix it?

I'm using [NX](https://nx.dev) by the way.

**Comment by Akxe** [comment, marked as answer]
You should only include Prisma from `@prisma/client` once. You should then create a Prisma instance to access the DB, that instance should be shared.

**Reply by cerpinn** [reply]
It's not once, I frequently import types from prisma client.

**Reply by Akxe** [reply]
True, I do not have this problem and I too use NX, to simply fix your issue you could re-export all types of `@prisma/client`.

**Reply by cerpinn** [reply]
Yeah it's honestly not big of a deal. I just thought maybe there's some sort of config that I don't know about to make it work.

But again, it's whatever.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
